managed/ui/src/components/config/Security/KeyManagementConfigurationContainer.js

```diff
@@ -32,26 +33,35 @@ const mapDispatchToProps = (dispatch) => {
     fetchKMSConfigList: () => {
       return dispatch(fetchAuthConfigList()).then((response) =>
         dispatch(fetchAuthConfigListResponse(response.payload))
-      );
+      )
+      .catch((err) => {
+        toast.error("Error occured while fetching config.")
```

Comment:
change to single quote and add `;` at the end pls

Reply:
Done